guide/interactions/slash-commands.md

```diff
@@ -110,6 +113,27 @@ const data = new SlashCommandBuilder()
 			.setDescription('The input to echo back')
 			.setRequired(true));
 ```
+:::
+
+::: code-group-item JSON
+```js
+const data = {
+	name: 'echo',
+	description: 'Replies with your input!',
+	options: [
+		{
+			name: 'input',
+			description: 'The input to echo back',
+			type: 'STRING',
```

Comment:
just naming one out of the many, If you're actually going to use json, look at the docs, they don't take in strings, they take in numbers, the enum values
sure in v13, the applicationcommandmanager.create takes in the string i think but aren't you people discouraging this way of registering??

Reply:
Oh i completely missed that, you are right, the raw api takes numbers